jobs/statsd/templates/config/config.js.erb

```diff
@@ -99,7 +99,7 @@ Optional Variables:
 
 */
 {
-  graphitePort: <%= p('carbon.cache.line_receiver_port') %>
+  graphitePort: <%= p('carbon_cache_line_receiver_port') %>
```

Comment:
Can you explain the reason for renaming? Wouldn't it be preferable to keep the namespacing?